pkg/feature/storage/v2/sql/flag_trigger/list_flag_trigger.sql

```diff
@@ -0,0 +1,13 @@
+SELECT id,
+       feature_id,
+       environment_namespace,
```

Comment:
We don't need to get the `environment_namespace`.

Reply:
Sure, we can delete this column from list sql.